Documentation/operations/upgrade.rst

```diff
@@ -312,6 +312,18 @@ Annotations:
 1.14 Upgrade Notes
 ------------------
 
+Added Metrics
+~~~~~~~~~~~~~
+
+* ``cilium_policy_change_total``
+
+Deprecated Metrics
+~~~~~~~~~~~~~~~~~~
+
+* ``cilium_policy_import_errors_total`` is deprecated. Please use
```

Comment:
thanks a lot :100:

Reply:
nit:

Ideally, we try our best to leverage new metrics in dashboard/alert once they are introduced as these things are easierly overlooked.

https://github.com/cilium/cilium/blob/dcc458b5f423d5a22ab53b7888c69eb1d675d88b/examples/kubernetes/addons/prometheus/monitoring-example.yaml#L6031

https://github.com/cilium/cilium/blob/1626543f4164294d4afc576daf390c6f95a9de6a/examples/kubernetes/addons/prometheus/files/grafana-dashboards/cilium-dashboard.json#L5679

Reply:
Done